### PR TITLE
Task#88 dashboard receive

### PR DESCRIPTION
**Pull request by NemTechCN**
image

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -84,6 +84,8 @@ export const Message = {
     OPERATION_SUCCESS: 'successful_operation',
     UPDATE_SUCCESS: 'update_completed',
     NODE_CONNECTION_SUCCEEDED: 'Node_connection_succeeded',
+    PLEASE_ENTER_THE_CORRECT_NUMBE: 'Please_enter_the_correct_numbe',
```
Comment (decentraliser) [inline src/config/index.ts]: Typo number

File: src/language/en-US.ts [hunk 2]
```diff
@@ -686,6 +686,11 @@ export const en_US = {
     all_unchecked: 'Uncheck all',
     Display_expired_mosaic:'Display expired mosaics',
     Hide_expired_mosaic:'Hide expired mosaics',
+    Please_enter_notes:'Please enter remarks',
+    Invoice:'Invoice',
+    Please_enter_the_amount_of_transfer:'Please enter the amount of transfer',
```
Comment (decentraliser) [inline src/language/en-US.ts]: Number

File: src/language/en-US.ts [hunk 3]
```diff
@@ -686,6 +686,11 @@ export const en_US = {
...
+    Notes_should_not_exceed_25_character:'Notes should not exceed 25 character',
```
Comment (decentraliser) [inline src/language/en-US.ts]: Characters

File: src/views/monitor/monitor-invoice/MonitorInvoice.vue [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    </div>
+    <CollectionRecord :transactionType="TransferType.RECEIVED"></CollectionRecord>
+
+    <div class="modal">
+      <div class="modal_title">{{$t('set_amount')}}</div>
+      <div class="asset flex_center">
+        <span class="title">{{$t('asset_type')}}</span>
+        <span class="value radius flex_center">
+          <Select :placeholder="$t('please_choose_mosaic')" class="asset_type" v-model="formItem.mosaicHex">
```
Comment (decentraliser) [inline src/views/monitor/monitor-invoice/MonitorInvoice.vue]: Shouldn't it accept free text input as well?

File: src/language/en-US.ts (hunk 2 above)
Comment (NemTechCN) [reply src/language/en-US.ts]: Please enter the number of transfer?

Comment (decentraliser) [reply src/language/en-US.ts]: This is a fat finget, forget about it

Should be "please enter the transfer amount" but I can fix the English translation

File: src/config/index.ts (hunk 1 above)
Comment (NemTechCN) [reply src/config/index.ts]: fixed

File: src/language/en-US.ts (hunk 3 above)
Comment (NemTechCN) [reply src/language/en-US.ts]: fixed

File: src/views/monitor/monitor-invoice/MonitorInvoice.vue (hunk 4 above)
Comment (NemTechCN) [reply src/views/monitor/monitor-invoice/MonitorInvoice.vue]: fixed

**Comment by NemTechCN** [comment]
image

**Pull request merged**